local-cli/generator-macos/templates/macos/HelloWorld-iOS/Images.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -39,11 +29,6 @@
       "idiom" : "iphone",
       "size" : "60x60",
       "scale" : "3x"
-    },
-    {
-      "idiom" : "ios-marketing",
-      "size" : "1024x1024",
-      "scale" : "1x"
```

Comment:
Will double check that these were removed from the iOS target upstream.

Reply:
Looks like it https://github.com/facebook/react-native/blob/v0.62.2/template/ios/HelloWorld/Images.xcassets/AppIcon.appiconset/Contents.json